Assets/Project/Modules/VFX/Anchor/Throw/Scripts/AnchorThrowHeadFollower.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using UnityEngine;
+
+namespace Project.Modules.VFX.Anchor.Throw.Scripts
+{
+    public class AnchorThrowHeadFollower : MonoBehaviour
+    {
+        [Header("REFERENCES")] 
+        [SerializeField] private Transform _anchorMoveRotate;
+        
+        private float _fixedHeight;
+
+        public void OnEnable()
+        {
+            _fixedHeight = _anchorMoveRotate.position.y;
+        }
+
```

Comment:
I would add 2 functions:
`public void StartFollowing(){
   gameObject.SetActive(true);
   UpdateFixedHeight();
}`

and

`public void StopFollowing(){
   gameObject.SetActive(false);
}`

and remove the OnEnable() function

This way it is easier to track who is using this script and when.

Reply:
Done 👌